src/components/teasers/grid/teaser-grid.tsx

```diff
@@ -1,9 +1,21 @@
 import React from 'react';
 import styled from 'styled-components';
-import { up } from '../../breakpoint/breakpoint';
+import { up } from '../../style-utils/breakpoint';
+import { isNoneTouch } from '../../style-utils/is-touch';
 
 const ScrollContainer = styled.div`
-  overflow-x: auto;
+  overflow: hidden;
+
+  ${isNoneTouch()} {
+    overflow-x: scroll;
```

Comment:
why not auto? This always forces a scrollbar? I changed this before to auto because of this
![Screen Shot 2021-12-08 at 18 08 56](https://user-images.githubusercontent.com/1701755/145252213-2e8907ed-7997-4707-a4a7-d8b04eac3c5f.png)

Can you explain the problem with scroll between mobile & desktop again? I never had any special issues in my mind. Why change it across device types?

Reply:
The big issue is the different scroll bar handling between touchpad (on Mac) and a real mouse. Because the browser adds some padding to permanent scroll bars. I was probably just lost in the issue, but you are right: `overflow-x: auto` works when it's added to the mobile layout. But the extra `padding-bottom` on mobile is needed for touchpads, as the scrollbar will otherwise flow over the arrow.

Reply:
Okay as said, I have problems reproducing it. Maybe you can demonstrate it to me at some time. Good for now!